recipes/cn-cbor/all/conandata.yml

```
@@ -0,0 +1,4 @@
+sources:
+  "20200326":
```

```suggestion
  "20180730":
```

Use the date of commit

---

see above